src/others/vg_lite_tvg/vg_lite_matrix.c

```diff
@@ -103,13 +113,19 @@ vg_lite_error_t vg_lite_scale(vg_lite_float_t scale_x, vg_lite_float_t scale_y,
 
     /* Multiply with current matrix. */
     multiply(matrix, &s);
+
+#if VG_SW_BLIT_PRECISION_OPT
+    matrix->scaleX = matrix->scaleX * scale_x;
+    matrix->scaleY = matrix->scaleY * scale_y;
+#endif /* VG_SW_BLIT_PRECISION_OPT */
+
     return VG_LITE_SUCCESS;
 }
 
 vg_lite_error_t vg_lite_rotate(vg_lite_float_t degrees, vg_lite_matrix_t * matrix)
 {
     /* Convert degrees into radians. */
-    vg_lite_float_t angle = (vg_lite_float_t)(degrees / 180.0f * M_PI);
+    vg_lite_float_t angle = (degrees / 180.0f) * 3.141592654f;
```

Comment:
Please check if it generates compile warning here.

Reply:
Done